api/lib/application/assessment-results/index.js

```diff
@@ -14,7 +14,6 @@ exports.register = async (server) => {
                 securityPreHandlers.checkUserHasRoleSuperAdmin,
                 securityPreHandlers.checkUserHasRoleCertif,
                 securityPreHandlers.checkUserHasRoleSupport,
-                securityPreHandlers.checkUserHasRoleMetier,
```

Comment:
**question**
Cela n'a pas de rapport avec le code, mais je trouve que mettre la remarque sur la PR fait que l'on ne la voit pas toujours

Y a-t-il un cas où un rôle puisse ne pas être lié à un utilisateur (table `pix-admin-roles`) ?
Si ce n'est pas le cas, une contrainte `NOT NULL` peut être ajoutée en BDD pour prévenir l'insertion de données incorrecte par le code en cas de bug.

```
pix=# \d "pix-admin-roles" 
                                       Table "public.pix-admin-roles"
   Column   |           Type           | Collation | Nullable |                   Default                   
------------+--------------------------+-----------+----------+---------------------------------------------
 id         | integer                  |           | not null | nextval('users_pix_roles_id_seq'::regclass)
 userId     | integer                  |           |          | 
 role       | character varying(255)   |           | not null | 
```